### PR TITLE
add alert and avatar components

### PR DESCRIPTION
**Pull request by Maianki**
Added following components : 
## Simple alert
Simple Alerts

## Alerts with icons
Alerts with icons

## Different types of Avatars
Different types of avatars

Preview Link :- 
https://shuttle-ui-testing.netlify.app/docs.html

Feedbacks appreciated. And thanks for reviewing my work 🙂.

### CODE REVIEW

**Comment by vrinda-mahajan** [comment]
LGTM👍

File: components/components-styles/alert.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    --danger-alert-color:#973e46;
+    --warning-alert-color:#a18d51;
+    --info-alert-color:#145d6c;
+    --primary-alert-bgcolor:#cfe2ff;
+    --secondary-alert-bgcolor:#e2e3e5;
+    --success-alert-bgcolor:#d1e7dd;
+    --danger-alert-bgcolor:#f8d7da;
+    --warning-alert-bgcolor:#fff3cd;
+    --info-alert-bgcolor:#cff4fc;
+}
```
Comment (Dev-AkshatGupta) [inline components/components-styles/alert.css]: Change the color names to the capital letters Would become easy for you later on to read and react accordingly.
though great UI

Comment (Dev-AkshatGupta) [reply components/components-styles/alert.css]: Else everything looks amazing Keep up the good work. :)

Comment (Maianki) [reply components/components-styles/alert.css]: Okay. Will change that.

**Comment by Maianki** [comment]
> LGTM👍

Thanks for reviewing.

File: components/components-styles/alert.css (hunk 1 above)
Comment (Maianki) [reply components/components-styles/alert.css]: > Change the color names to the capital letters Would become easy for you later on to read and react accordingly. though great UI

Hi, Akshat I have made the above changes please do a review again.

Comment (sainath7878) [reply components/components-styles/alert.css]: LGMT.

Comment (sainath7878) [reply components/components-styles/alert.css]: Make sure the contrast ratio is good. It's a bit low for warning alert.

**Pull request merged**